flocker/control/test/test_httpapi.py

```diff
@@ -23,19 +26,44 @@ class APITestsMixin(object):
     """
     Integration tests for the Dataset Manager API.
     """
-    def test_version(self):
+    def initialize(self):
```

Comment:
I guess this isn't `setUp` because `buildIntegrationTests` doesn't have a cooperative `setUp`?  Sad.  Wish we had proper fixture support.  Oh well.